src/styles/index.ts

```diff
@@ -3359,13 +3359,8 @@ const styles = (theme: ThemeColors) =>
         },
 
         workspaceThreeDotMenu: {
-            marginLeft: 84,
-        },
-
-        workspaceListBadge: {
-            flexDirection: 'column',
-            justifyContent: 'flex-start',
-            marginTop: 6,
+            justifyContent: 'flex-end',
+            width: 124,
```

Comment:
Can you share some mobile screenshots of the workspaces page where the workspace names are quite long? If I recall correctly from before, setting a fixed width here means that when there is no badge shown, we still lose a lot of horizontal space for the workspace name to display.

Reply:
Here is how it looks:
<img width="410" alt="image" src="https://github.com/Expensify/App/assets/50919443/2b68ecb8-32fb-462b-8d56-d3db06014465">

Btw, the above style is applied to a large screen.

Reply:
Ah okay, thanks for confirming - that looks good on mobile.